fedimint-dbtool/src/main.rs

```diff
@@ -89,9 +89,21 @@ fn print_kv(key: &[u8], value: &[u8]) {
     println!("{} {}", key.to_hex(), value.to_hex());
 }
 
+fn handle_version_hash_command() {
+    let mut args = std::env::args();
+    if let Some(ref arg) = args.nth(1) {
+        if arg.as_str() == "version-hash" {
+            println!("{}", env!("FEDIMINT_BUILD_CODE_VERSION"));
+            std::process::exit(0);
+        }
+    }
+}
+
```

Comment:
Wondering if this fn could live somewhere more central? There must be similar code all over the place.

Reply:
Good callout :+1:

`fedimint_core::util` seemed like a natural fit, but let me know if you think there's a better place.

https://github.com/fedimint/fedimint/pull/4232/commits/26e2e56ef1fa8c3832ffc44f35a9a0c745a9a3d0

Verified after building the binaries:

```
nix@lenovo-m75q:~/fedimint$ ./target/debug/fedimint-dbtool version-hash
5a941b7339f332b3000000008d37e4f8e611f4
nix@lenovo-m75q:~/fedimint$ ./target/debug/fedimintd version-hash
5a941b7339f332b3000000008d37e4f8e611f4
nix@lenovo-m75q:~/fedimint$ ./target/debug/gateway-cln-extension version-hash
5a941b7339f332b3000000008d37e4f8e611f4
nix@lenovo-m75q:~/fedimint$ ./target/debug/gatewayd version-hash
5a941b7339f332b3000000008d37e4f8e611f4
nix@lenovo-m75q:~/fedimint$ ./target/debug/fedimint-cli version-hash
5a941b7339f332b3000000008d37e4f8e611f4
nix@lenovo-m75q:~/fedimint$ ./target/debug/devimint version-hash
5a941b7339f332b3000000008d37e4f8e611f4
```